core/server/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -610,8 +609,12 @@ private void processWorkerRemovedBlocks(MasterWorkerInfo workerInfo,
     for (long removedBlockId : removedBlockIds) {
       MasterBlockInfo masterBlockInfo = mBlocks.get(removedBlockId);
       if (masterBlockInfo == null) {
-        LOG.warn("Worker {} removed block {} but block does not exist.", workerInfo.getId(),
-            removedBlockId);
+        LOG.warn("Worker {} informs the removed block {}, but block metadata does not exist"
```

Comment:
Similarly, the eviction operations from Workers should also be catched at: https://github.com/Alluxio/alluxio/blob/master/core/server/src/main/java/alluxio/master/block/BlockMaster.java#L619